web/src/unread_ui.js

```diff
@@ -13,20 +13,45 @@ import * as topic_list from "./topic_list";
 import * as unread from "./unread";
 import {notify_server_messages_read} from "./unread_ops";
 
-let user_closed_mark_as_read_turned_off_banner = false;
-export function hide_mark_as_read_turned_off_banner() {
+let user_closed_unread_banner = false;
+
+export function update_unread_banner() {
+    $("#mark_as_read_turned_off_banner").html(render_mark_as_read_turned_off_banner());
+
+    $("#mark_view_read").on("click", () => {
```

Comment:
Calling this more than once is dangerous -- one can end up with multiple copies of the click handler running on a single click. We should leave this in `initialize`, possibly with a `$("body").on("click", "#mark_view_read"` structure.

Reply:
Yeah, you'll notice we use that style of click handler hanging off body a lot in the codebase, because it's more robust in this way.